app/main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list should be a class variable, not an instance variable. It should be defined as `Animal.alive = []` outside of the class definition to ensure it is shared across all instances.